samples/openapi3/client/petstore/typescript/builds/browser/types/ObjectParamAPI.ts

```diff
@@ -23,12 +23,14 @@ export interface PetApiAddPetRequest {
 export interface PetApiDeletePetRequest {
     /**
      * Pet id to delete
+     * Defaults to: undefined
```

Comment:
There are a bunch of these. These are technically correct, however read a bit odd especially if the parameter is a required parameter. As mustache is logicless we can't easily exclude them with an undefined check. We could possibly introduce a new template var that is only populated if its not `"undefined"`.